queryfilter/tests/test_datetimefilter.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import absolute_import
+
+import pytest
+import dateutil.parser
+from test_app.models import DatetimeFilterTestingModel
+from ..datetimefilter import DatetimeRangeFilter
+
+
+ordered_triple_test_date_set = (
+    ("2016-12-29T00:00:00+08:00", "2016-12-30T00:00:00+08:0", "2016-12-31T00:00:00+08:0"),  # in three day
+    ("2016-12-31T00:00:00+08:00", "2016-12-31T01:00:00+08:0", "2016-12-31T02:00:00+08:0"),  # in one day
+    ("2016-12-31T23:00:00+08:00", "2017-01-01T00:00:00-08:0", "2018-12-31T02:00:00+08:0"),  # across year
+)
+
+date_time_parametrize = pytest.mark.parametrize("dates, test_data, queryset",
+                                                zip(ordered_triple_test_date_set,
+                                                    ordered_triple_test_date_set,
+                                                    ordered_triple_test_date_set,
+                                                    ),
+                                                indirect=['test_data', 'queryset'])
+
+
+@pytest.fixture
+def test_data(request):
+    data = [{FIELD_NAME: date} for date in request.param]
+    return data
+
+
+@pytest.fixture
+def queryset(request):
+    data = test_data(request)
+
+    for datum in data:
+        DatetimeFilterTestingModel.objects.create(**{
+            FIELD_NAME: dateutil.parser.parse(datum[FIELD_NAME])
```

Comment:
I'm thinking just datetime will be fine, since we're specifically using with DatetimeFilterTestingModel?

Reply:
ah ... never mind, `cause you're using it cross functions 😜